install-worker.sh

```diff
@@ -173,19 +173,30 @@ sudo rm -rf \
     $TEMPLATE_DIR  \
     /var/cache/yum
 
+unset IFS
+# Kill dhclient to not cache search domains
+for DHPID in $(/usr/sbin/pidof dhclient) ; do
```

Comment:
dhclient is started by systemd so it will get restarted if you just kill it. Maybe `sudo sysctl stop network` instead?

Reply:
I double checked, and this isn't actually required.